scripts/patch.js

```diff
@@ -91,7 +91,7 @@ async function addFiles (localPath) {
   const patchRoot = await addFiles(PATCH_SRC)
   console.log(`Patch root is ${patchRoot}`)
 
-  const distRoot = await ipfs.dns(DIST_DOMAIN)
```

Comment:
:information_source: `dns` is deprecated, switching to `resolve` avoids having to deal with bug described in https://github.com/ipfs/kubo/pull/10212